cmd/internal/cli/build_darwin.go

```diff
@@ -31,12 +31,13 @@ func runBuild(cmd *cobra.Command, args []string) {
 		sylog.Fatalf("Only remote builds are supported on this platform")
 	}
 
-	bc, lc, err := getBuildAndLibraryClientConfig(buildArgs.builderURL, buildArgs.libraryURL)
+	bc, lc, kc, err := getServiceConfigs(buildArgs.builderURL, buildArgs.libraryURL, buildArgs.keyServerURL)
 	if err != nil {
 		sylog.Fatalf("Unable to get builder and library client configuration: %v", err)
```

Comment:
Maybe just generalize this?

```suggestion
		sylog.Fatalf("Unable to get service configuration: %v", err)
```